pkg/resourceinterpreter/default/native/aggregatestatus.go

```diff
@@ -554,3 +556,34 @@ func aggregatePodDisruptionBudgetStatus(object *unstructured.Unstructured, aggre
 	pdb.Status = *newStatus
 	return helper.ToUnstructured(pdb)
 }
+
+func aggregateHorizontalPodAutoscalerStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
```

Comment:
Please add some  unit tests to cover this function.

Reply:
done